backend/index.ts

```diff
@@ -1,21 +1,26 @@
-import express from 'express';
-import dotenv from 'dotenv';
-import cors from 'cors';
+import express from "express";
+import dotenv from "dotenv";
+import cors from "cors";
+import mongoose from "mongoose";
 
 dotenv.config();
 
 const app = express();
 const port = process.env.PORT || 5000;
 
 app.use(express.json());
-app.use(express.urlencoded({extended: false}));
+app.use(express.urlencoded({ extended: false }));
 
 app.use(cors());
 
-app.get('/health', (req, res) => {
-    res.send("Express server is working correctly!");
-})
+mongoose.connect("mongodb://localhost:27017/CalendarDB").then(() => {
```

Comment:
Should define the mongodb url in the environment variable.
It should be changed in different stages of the product so it should not be directly embedded in the code.